src/main/java/org/dcsa/bkg/model/transferobjects/BookingCancellationRequestTO.java

```diff
@@ -1,14 +1,18 @@
 package org.dcsa.bkg.model.transferobjects;
 
 import lombok.Data;
-import org.dcsa.core.events.model.enums.DocumentStatus;
+import org.dcsa.core.events.model.enums.ShipmentEventTypeCode;
+import org.dcsa.core.validator.EnumSubset;
 
 import javax.validation.constraints.NotNull;
 
+import static org.dcsa.core.events.model.enums.ShipmentEventTypeCode.BOOKING_DOCUMENT_STATUSES;
+
 @Data
 public class BookingCancellationRequestTO {
 
 	@NotNull(message = "DocumentStatus is required")
-	private DocumentStatus documentStatus;
+	@EnumSubset(anyOf = BOOKING_DOCUMENT_STATUSES)
```

Comment:
It might make sense to use the @EnumSubset validation to further restrict this. Since in the BookingCancellationRequestTO this can only have one value: CANC

this could also be used to perform this validation check: https://github.com/dcsaorg/DCSA-BKG/pull/81/files#diff-f0cb7b5ac41a7ee88163e60403c236eb4e59e16a571c8a8641a9fa8deeb4f1a5R56

Reply:
Implemented this suggestion - though I find it redundant to require an input field with only one specific value allowed.